### PR TITLE
Add a basic informational site

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Rather than jumping straight into an application that manages login/email/etc. let's add a basic static site with some basic Bootstrap theming to kick off discussion on the content and have something we could point people to today.

This is something like a 0.1 alpha; we would not be actively promoting this PR.

What I'd love help with here:
- code review
- design review (and design help)
- review on the wording
- thoughts on what else we'd need to have something that's not totally embarrassing to have on the internet

Thanks everyone :heart:!

### CODE REVIEW

**Comment by andrew** [comment]
@MikeMcQuaid some screenshots would be ✨

**Comment by MikeMcQuaid** [comment]
@andrew Good idea:

screen shot 2017-02-14 at 17 17 36
screen shot 2017-02-14 at 17 17 41
screen shot 2017-02-14 at 17 17 48
screen shot 2017-02-14 at 17 17 52

I'll want to get this less LOL BOOTSTRAP before shipping.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,16 +1,33 @@
 <!DOCTYPE html>
-<html>
+<html class="container">
```
Comment (Charlotteis) [inline app/views/layouts/application.html.erb]: can we add a `lang="en"` on this `html` tag?

**Comment by Charlotteis** [comment]
I _think_ that navigation bar _might_ fail colour contrasts tests. Can't take a look myself right now, but you can test it using https://khan.github.io/tota11y/ :D

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (MikeMcQuaid) [inline app/views/layouts/application.html.erb]: Done 👍

**Comment by MikeMcQuaid** [comment]
@Charlotteis Great catches, thanks! Have addressed them all and added that tool to use in development mode.

**Comment by MikeMcQuaid** [comment]
> I think that navigation bar might fail colour contrasts tests.

Just to be super explicit: you were right so I switched this to use an inverse navbar.

**Comment by andrew** [comment]
This looks like a great start, merge when ready!

**Comment by Charlotteis** [comment]
Good work @MikeMcQuaid, I like the addition of tota11y in dev!

**Pull request merged**